saleor/core/utils/metadata_manager.py

```diff
@@ -73,3 +74,12 @@ def create_from_graphql_input(
     return MetadataItemCollection(
         [MetadataItem(item.key, item.value) for item in items]
     )
+
+
+def method_metadata_is_valid(metadata: Any) -> bool:
```

Comment:
What is `method`? 🤔  It looks like after moving to different file, the function lost the context. Maybe worth to add it 🤔

Reply:
changed to `metadata_is_valid`